### PR TITLE
iOS: Bump SDK and UXKit, remove fixed version number

### PR DESCRIPTION
**Pull request by yganu**

### CODE REVIEW

File: voxeet-uxkit-reactnative.podspec [hunk 1]
```diff
@@ -9,14 +9,14 @@ Pod::Spec.new do |spec|
...
+  spec.platform = :ios, "12.0"
   spec.swift_version = "5.5.2"
   spec.source = { :git => "https://github.com/voxeet/voxeet-uxkit-reactnative.git", :tag => "v#{spec.version}" }
   spec.source_files  = "ios/**/*.{h,m}"
   spec.framework = "UIKit"
   spec.dependency "React"
-  spec.dependency "VoxeetUXKit", "1.6.1"
-  spec.dependency "VoxeetSDK", "3.3.1"
+  spec.dependency "VoxeetUXKit", "~>1.0"
+  spec.dependency "VoxeetSDK", "~>3.0"
```
Comment (FabienLavocat) [inline voxeet-uxkit-reactnative.podspec:19]: Do we need this `VoxeetSDK` dependency as it is already a dependency of `VoxeetUXKit`?
https://github.com/voxeet/voxeet-uxkit-ios/blob/master/VoxeetUXKit.podspec#L14

Comment (yganu) [reply voxeet-uxkit-reactnative.podspec]: You are right, I've just updated the PR

**Review by ezugaj — APPROVED** [review]
While generally I agree the approach, what happens if UXKit moves to next swift version?

**Comment by yganu** [comment]
> 

We can remove the hardcoded swift version, user should be able to choose the version that they need.
Additionally it is already hardcoded in uxkit, I think it also has to be removed:
https://github.com/voxeet/voxeet-uxkit-ios/blob/master/VoxeetUXKit.podspec

**Review by FabienLavocat — APPROVED** [review]

**Pull request merged**